### PR TITLE
Fix recursion ability count version

### PR DESCRIPTION
**Pull request by newfrenchy83**

### CODE REVIEW

File: wml_test_schedule [hunk 1]
```diff
@@ -383,6 +383,9 @@
 9 four_cycle_recursion_branching
 9 four_cycle_recursion_by_id
 9 four_cycle_recursion_by_tagname
+9 abilitie_cycle_recursion_by_id
+9 filter_self_abilitie_cycle_recursion_by_id
```
Comment (stevecotton) [inline wml_test_schedule:387]: The singular of "abilities" is "ability".

Comment (newfrenchy83) [reply wml_test_schedule]: fixed

Comment (stevecotton) [reply wml_test_schedule]: Two of the filenames need fixing too.

Comment (newfrenchy83) [reply wml_test_schedule]: fixed

File: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_filter_self_recursion_by_id.cfg [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [filter_self]ability_id_active=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# Give Alice's leadership_test_recursion ability.
+# leadership_test_recursion active if itself active
```
Comment (stevecotton) [inline data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_filter_self_recursion_by_id.cfg:9]: Give Alice a leadership_test_recursion ability, which is active only if it is itself active.

**Review by stevecotton — APPROVED** [review]
Ready to merge, except for minor grammar in the test descriptions.

File: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_filter_self_recursion_by_id.cfg (hunk 2 above)
Comment (newfrenchy83) [reply data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_filter_self_recursion_by_id.cfg]: corrected

File: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/adjacent_abilities_recursion_by_id.cfg [hunk 3]
```diff
@@ -0,0 +1,62 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [affect_adjacent][filter]ability_id_active=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# Give Alice's and Bob leadership_test_recursion ability.
+# leadership_test_recursion active if adjacent unit has leadership_test_recursion active
```
Comment (stevecotton) [inline data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/adjacent_abilities_recursion_by_id.cfg:9]: Give Alice and Bob a leadership_test_recursion ability, which is active only if an adjacent unit also has it active.

File: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_recursion_by_id.cfg [hunk 4]
```diff
@@ -0,0 +1,58 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [filter]ability_id_active=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# Give Alice's leadership_test_recursion ability.
+# leadership_test_recursion active if itself active
```
Comment (stevecotton) [inline data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_recursion_by_id.cfg:9]: Give Alice and Bob a leadership_test_recursion ability, which is active only if it is itself active.

File: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/adjacent_abilities_recursion_by_id.cfg (hunk 3 above)
Comment (newfrenchy83) [reply data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/adjacent_abilities_recursion_by_id.cfg]: fixed

File: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_recursion_by_id.cfg (hunk 4 above)
Comment (newfrenchy83) [reply data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_recursion_by_id.cfg]: fixed

**Pull request merged**